target/classes/templates/start.html

```diff
@@ -11,15 +11,20 @@
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
 
 </head>
+
+<header>
+
+    <div th:replace="fragments/topnav :: navbar"></div>
+
+</header>
 <body>
-<h1>Fundraiser Manager</h1>
     <!---<div th:replace="fragments/topnav :: navbar">
     </div> -->
-    <div class="topnav">
+    <!--<div class="topnav">
         <a href="start.html">Home</a>
         <a href="members.html">Members</a>
         <a href="fundraisers.html">Fundraiser</a>
-    </div>
+    </div> -->

```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.